nym-api/migrations/20240524101449_monitor_v2_tables.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE mixnode_details_v2
+(
+    id       INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    mix_id   INTEGER NOT NULL UNIQUE,
+    owner    VARCHAR NOT NULL,
```

Comment:
mmm. future merge conflicts : )

if you want to make my life easier, could you add `node_id` to `gateway_details` and remove `owner` column from both gateways and mixnodes?
if that's too much work on your end (because it's all over the place), don't worry. I've already handled migration from the old `mixnode_details` and `gateway_details` so re-applying them shouldn't be too much of a problem